src/features/review/reviewSlice.js

```diff
@@ -207,6 +208,9 @@ export const reviewSlice = createSlice({
       })
       .addCase(deleteImagesSuccess, (state, { payload }) => {
         state.workingImages = state.workingImages.filter(({ _id }) => !payload.includes(_id));
+      })
+      .addCase(deleteImageTaskSuccess, (state, { payload }) => {
+        state.workingImages = state.workingImages.filter(({ _id }) => !payload.includes(_id));
       });
```

Comment:
Do we also want to add a case for the delete-by-filters-task here? e.g. `.addCase(deleteImagesByFilterTaskSuccess, ...)`

Reply:
related: https://github.com/tnc-ca-geo/animl-frontend/pull/250#discussion_r1828474326

Reply:
Ah yup got it!